### ISSUE TITLE
ERROR: Exception thrown (in game.py)

### ISSUE THREAD

**Issue by agboom** [wackerl91/luna#24]
The add-on fails to show the list of games. The following is printed in the log:

```
20:16:12 7241.391113 T:1548743712   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!      
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: object of type 'NoneType' has no len()
                                            Traceback (most recent call last):          
                                              File "/home/osmc/.kodi/addons/luna-master/addon.py", line 107, in <module>
                                                plugin.run()                            
                                              File "/home/osmc/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 332, in run
                                                items = self._dispatch(self.request.path)  
                                              File "/home/osmc/.kodi/addons/script.module.xbmcswift2/lib/xbmcswift2/plugin.py", line 306, in _dispatch
                                                listitems = view_func(**items)               
                                              File "/home/osmc/.kodi/addons/luna-master/addon.py", line 76, in show_games
                                                return plugin.finish(game_controller.get_games_as_list(), sort_methods=['label'])                                       
                                              File "/home/osmc/.kodi/addons/luna-master/resources/lib/controller/gamecontroller.py", line 77, in get_games_as_list
                                                self.get_games()                         
                                              File "/home/osmc/.kodi/addons/luna-master/resources/lib/controller/gamecontroller.py", line 35, in get_games
                                                storage[game_name] = self.scraper_chain.query_game_information(game_name)
                                              File "/home/osmc/.kodi/addons/luna-master/resources/lib/scraper/scraperchain.py", line 32, in query_game_information
                                                game.merge(next_game)     
                                              File "/home/osmc/.kodi/addons/luna-master/resources/lib/model/game.py", line 32, in merge
                                                elif len(other.plot) > len(self.plot):
                                            TypeError: object of type 'NoneType' has no len()
                                            -->End of Python script error report<--
20:16:12 7241.555664 T:1957790256   ERROR: GetDirectory - Error getting plugin://script.luna/games                                            
20:16:12 7241.556152 T:1957790256   ERROR: CGUIMediaWindow::GetDirectory(plugin://script.luna/games) failed     
```

The changelog.txt in `home/osmc/.kodi/addons/luna-master/changelog.txt` contains:

```
v0.3.0-alpha:
-- during alpha changelog won't be updated, except version number
```

**Comment by wackerl91** [comment]
Thanks for letting me know! This will be fixed in the upcoming version (should be done by Wednesday). Meanwhile it should work if you disable the scraper feature completely in add-on settings > scraper settings > disable experimental scraper feature; you'll miss out on game information though and Luna will just show the name of your games.
If for whatever reason this workaround doesn't work please let me know here and I'll try to find another solution for you!

**Comment by agboom** [comment]
Thanks! Turning off the scraper works!

**Comment by wackerl91** [comment]
Glad this works for you! :) I'll update this issue accordingly once the new release is ready - it will likely still contain some bugs here and there (just found another one :( ), so please don't expect it to work perfectly :)

**Comment by wackerl91** [comment]
This should be resolved in V0.4.0.
If a similar error occurs again please open a new issue. Thanks!
